porter/v1/service.proto

```diff
@@ -128,3 +131,20 @@ message GPU {
   int32 gpu_cores_nvidia = 2;
   //other types of gpus can be added here
 }
+
+// Tolerations are any tolerations that should be applied to the application
+message Toleration {
+  // key is the key of the toleration
+  string key = 1;
+  // operator is the operator of the toleration
+  string operator = 2;
+  // value is the value of the toleration
+  string value = 3;
+  // effect is the effect of the toleration
+  string effect = 4;
+}
```

Comment:
would we want to abstract out the kubernetes specific terms to something general? And they just select the node_group, and we turn this into the `key, operator, value, effect` in CCP?